client/app.jsx

```diff
@@ -72,6 +73,28 @@ export default class App extends React.Component {
     this.setState({ watchlistOpen: false });
   }
 
+  deleteFromWatchlist(episode) {
+    const deleteId = parseInt(episode, 10);
+    fetch(`/api/watchlist/${deleteId}`, {
+      method: 'DELETE',
+      body: JSON.stringify(episode)
+    })
+      .then(episode => {
+        let entryToDelete;
+        const watchlist = this.state.watchlist;
```

Comment:
Be sure to `.slice()` your array before modifying it. A new variable for an object or array does not make a copy.